vm/qemu/qemu.go

```diff
@@ -750,6 +754,54 @@ func (inst *instance) Diagnose(rep *report.Report) ([]byte, bool) {
 	return ret, false
 }
 
+func needsRegisterInfo(rep *report.Report) bool {
```

Comment:
I think it'd be better to list the report types for which we do not want register info and enable it for everything else by default (as you already do).